.github/workflows/_conformance_tests.yaml

```diff
@@ -49,6 +55,16 @@ jobs:
         env:
           JUNIT_REPORT: "conformance-tests.xml"
           KONG_TEST_EXPRESSION_ROUTES: ${{ matrix.expression_routes }}
+          TEST_KONG_KIC_MANAGER_LOG_OUTPUT: ${{ inputs.log-output-file }}
+      
+      # REVIEW: upload logs only when test failed?
```

Comment:
I'm okay with this. If all the jobs have passed, I don't think there's any specific reason to upload or download the logs.

Reply:
done.